controlplane/test/label.test.ts

```diff
@@ -324,6 +332,10 @@ describe('Labels', (ctx) => {
         keycloakClient,
         platformWebhooks,
         webBaseUrl,
+        slack: {
```

Comment:
Can we handle it optionally in the code?

Reply:
You mean make it optional and handle it while using it?

Reply:
Yes

Reply:
Done